src/Mod/BIM/bimcommands/BimMaterial.py

```diff
@@ -522,6 +520,11 @@ def rescan(self, rebuild=False):
                     i.setFlags(i.flags() | QtCore.Qt.ItemIsEditable)
                     if o.Name == name:
                         self.dlg.matList.setCurrentItem(i)
+            if hasattr(self.dlg, "buttonMergeDupes"):
+                if len(self.dlg.materials) > 1:
```

Comment:
Above you test what seems to be the same thing using the opposite test: `len(self.dlg.materials) < 2` -- I'd unify and use the same test in both places (I don't care which you use).